tests/test_utils/tests.py

```diff
@@ -4,13 +4,13 @@
 import unittest
 import warnings
 
+from django import forms
```

Comment:
@timgraham Thank you for reviewing this. I don't understand what is meant "Please keep cleanups like this in a separate commit or pull request". I know it affects previous code but the alternative would have been to import `Form`, `CharField` and `IntegerField` from `django.forms` individually. Can you kindly instruct and what should be done exactly? Please bear with me as this is my first-ever PR.